docs: update Workbench session token section to reflect UI flow (#1018)
fusedio/fused-docs: docs/guide/data-input-outputs/export-api/securing-shared-tokens.mdx

### From Workbench

> Workbench can generate session tokens directly from the UI. When you share a canvas with **team-only** access, click the **Share** button and select the share link option. Under the team section, you'll see a **Session token** option where you can choose an expiration duration and generate a token — no code or service account required.

Reviewer: Shorten this and:
Explain what Workbench does: i.e. Team scoped Canvas require a shared session token. 
So Workbench automatically creates a session token for you for each individual UDF you might want to share

> Session tokens are **UDF-bound** — each UDF on a canvas can have its own session token with a different expiration time. The generated URL will look like:

Reviewer: Not true, change this

Could be info tip box: These are created for the whole canvas

### From Workbench

Team-scoped canvases require a session token for external access. When you click the **Share** button, Workbench automatically generates a session token for each UDF you want to share — just choose an expiration duration. You can generate multiple session tokens with different expiration times, and each token will expire independently.

:::info

> Session tokens are created for the **whole canvas**. Every UDF on the canvas shares the same session token and expiration.

Reviewer: ```suggestion
Session tokens are created for the **whole canvas**. Any session token created can be used to access any other UDF & Widget in the same canvas.
```
